decimal.go

```diff
@@ -65,7 +71,10 @@ func newFromString(s string) (coeff *big.Int, exps []int64, err error) {
 
 // NewFromString creates a new decimal from s. It has no restrictions on
 // exponents or precision.
+//
+// REVIEW: Do we ever test error conditions here? I can imagine a number.
 func NewFromString(s string) (*Decimal, error) {
+	// REVIEW: why is this different from BaseContext.NewFromString(s)?
```

Comment:
Yep, the implementation was all I was referring to. I agree on the API standpoint.